presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -21,6 +21,9 @@
 
 namespace facebook::presto {
 
+// Setting this to 'true' makes configs modifiable via server operations.
+static constexpr std::string_view kMutableConfig{"mutable-config"};
```

Comment:
Would it be better to put the static into ConfigBase instead of making global to the facebook::presto namespace?

Reply:
Yes, we can do that.